src/main/java/org.wso2.apim.monetization/impl/workflow/StripeSubscriptionCreationWorkflowExecutor.java

```diff
@@ -175,6 +176,77 @@ public WorkflowResponse monetizeSubscription(WorkflowDTO workflowDTO, API api) t
         return execute(workflowDTO);
     }
 
+    @Override
+    public WorkflowResponse monetizeSubscription(WorkflowDTO workflowDTO, APIProduct apiProduct)
```

Comment:
Since there is no separation such as product and api in in API dev portal, we could add a single method for the workflow executor. WDYT

Reply:
However, there are some places where we need to separate the flow. Eg 
https://github.com/wso2-extensions/wso2-am-stripe-plugin/pull/9/files#diff-9fe56265548af83b1940a19175998d08R182

Also, WRT API product flow, we use different DAO methods.